snap/snapcraft.yaml

```diff
@@ -51,3 +71,4 @@ apps:
     - mount-observe
     - network-observe
     - system-observe
+    - proc-sys-kernel-random
```

Comment:
Do we need a `snap connect` addition to the README?

Reply:
Good catch!